
Compiler knows this is less than the sizeof of a domain name and throws
warnings which are treated as errors.


--- mDNSResponder-878.200.35/mDNSCore/DNSDigest.c.save	2019-05-02 09:25:26.147412708 +1000
+++ mDNSResponder-878.200.35/mDNSCore/DNSDigest.c	2019-05-02 09:25:28.119412751 +1000
@@ -1275,7 +1275,9 @@
 #define HMAC_OPAD   0x5c
 #define MD5_LEN     16
 
-#define HMAC_MD5_AlgName (*(const domainname*) "\010" "hmac-md5" "\007" "sig-alg" "\003" "reg" "\003" "int")
+static const domainname HMAC_MD5_AlgName = {
+	"\010" "hmac-md5" "\007" "sig-alg" "\003" "reg" "\003" "int"
+};
 
 // Adapted from Appendix, RFC 2104
 mDNSlocal void DNSDigest_ConstructHMACKey(DomainAuthInfo *info, const mDNSu8 *key, mDNSu32 len)
